### PR TITLE
CI/CD tool research

### PR DESCRIPTION
**Pull request by Flurb**

### CODE REVIEW

File: TOOLS_CHOICES.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Github Actions
+### Pros
+- Upcoming tool, really active in developing
+- Free tool for public Github repositories (which CoMPAS is)
+- Integrates perfectly with Github repositories
+
+### Cons
+- Software is proprietary
+- Not much experience within Alliander
```
Comment (Sander3003) [inline TOOLS_CHOICES.md:22]: Instead of Alliander exepriance, you can say adoption is growing

File: TOOLS_CHOICES.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+### Pros
+- Very much used CI/CD tool for Open Source projects
+
+### Cons
+- Not used within Alliander, so no experience or know-how
+- No advantages compared to the other evaluated tools
+
+## Jenkins
+### Pros
+- Lots of experience within Alliander
```
Comment (Sander3003) [inline TOOLS_CHOICES.md:34]: Well known in the industry.

File: TOOLS_CHOICES.md (hunk 1 above)
Comment (Flurb) [reply TOOLS_CHOICES.md]: Added!

File: TOOLS_CHOICES.md (hunk 2 above)
Comment (Flurb) [reply TOOLS_CHOICES.md:34]: Added!

**Review by Sander3003 — APPROVED** [review]

**Pull request merged**